servlet.py

```diff
@@ -1122,7 +1264,6 @@ def setupHandler(
     Handler.scaleMtLogs = scaleMtLogs
     Handler.verbosity = verbosity
     Handler.doc_pipe_sem = Semaphore(max_doc_pipes)
-
```

Comment:
Damn! I was experimenting what handlers are doing and removed it by mistake. Added again.